tensorflow_addons/layers/python/wrappers.py

```diff
@@ -131,25 +128,27 @@ def build(self, input_shape):
                 trainable=True,
                 aggregation=tf_variables.VariableAggregation.MEAN)
 
-            with ops.control_dependencies([self.layer.g.assign(
-                    self._init_norm(self.layer.v))]):
-                self._compute_weights()
+            self.layer.g.assign(self._init_norm(self.layer.v))
```

Comment:
We seems to drop the support for graph mode here, right? If yes, Is `@tf_test_util.run_all_in_graph_and_eager_modes` correct for test cases below?

Reply:
Yeah, the test coverage on this layer is not sufficient but it was added early to help build out the directory structure. I will revisit and add additional tests, but for now this recommended initialization is not required to achieve the decoupled kernel. I added a TODO to revisit, but I'd like to merge prior to our first package release so some of this outdated structure is not in the release.

Reply:
I agree.